modules/solid_mechanics/solid_mechanics.mk

```diff
@@ -1,5 +1,6 @@
 # NEML2
 ADDITIONAL_NEML2_DIRS += $(APPLICATION_DIR)/test/neml2
+ADDITIONAL_NEML2_DIRS += $(CURDIR)/neml2
```

Comment:
why did you choose `$(CURDIR)` here? I'm not sure this is correct (when using this module from another app).